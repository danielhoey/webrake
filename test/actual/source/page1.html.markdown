---
title: Page 1
---
# Kramdown test page

Testing the Kramdown filter
