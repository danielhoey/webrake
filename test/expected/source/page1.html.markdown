../../actual/source/page1.html.markdown