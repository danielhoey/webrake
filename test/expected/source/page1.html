---
title: Page 1
---
<h1 id="kramdown-test-page">Kramdown test page</h1>

<p>Testing the Kramdown filter</p>
